src/Microsoft.AspNet.Mvc.TagHelpers/AnchorTagHelper.cs

```diff
@@ -158,6 +165,7 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 if (Route == null)
                 {
                     tagBuilder = Generator.GenerateActionLink(
+                        viewContext: ViewContext,
```

Comment:
Don't really need to name this parameter. The rest are just named for the `string.Empty` case.